vtex/utils/transform.ts

```diff
@@ -302,7 +302,8 @@ export const toProduct = <P extends LegacyProductVTEX | ProductVTEX>(
     releaseDate,
     items,
   } = product;
-  const { name, ean, itemId: skuId, referenceId = [], kitItems } = sku;
+  const { name, ean, itemId: skuId, referenceId = [], kitItems, videos } = sku;
+  const video = nonEmptyArray(videos);
```

Comment:
Why `video`? It seems like you're just filtering the empty ones

Maybe `const nonEmptyVideos = ...`

Reply:
You are right